src/main/java/view/EditItemView.java

```diff
@@ -1,4 +1,205 @@
+                new JLabel("Min Temperature"), descriptonInputField);
+
+        inputs.add(nameInfo);
+        inputs.add(minTempInfo);
+        inputs.add(descriptionTempInfo);
+
+        imagePanel = new JPanel();
+
+        setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        saveChanges.addActionListener(
+                new ActionListener() {
+                    public void actionPerformed(ActionEvent evt) {
+                        if (evt.getSource().equals(saveChanges)) {
+                            GetClothingItemState currentState = getClothingItemViewModel.getState();
+
+                            updateController.execute(
+                                    currentState.getClothingItem()
+                            );
+                        }
+                    }
+                }
+        );
+
+        delete.addActionListener(
+                new ActionListener() {
+                    public void actionPerformed(ActionEvent evt) {
+                        if (evt.getSource().equals(delete)) {
+                            GetClothingItemState currentState = getClothingItemViewModel.getState();
+                            DeleteState deleteState = new DeleteState();
+                            deleteState.setDeletedItem(currentState.getClothingItem().getId());
+
+                            deleteViewModel.setState(deleteState);
+
+                            deleteController.execute(
+                                    currentState.getClothingItem().getId()
+                            );
+                        }
+                    }
+                }
+        );
+
+        back.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent evt) {
+                if (evt.getSource().equals(back)) {
+                    viewManagerModel.setActiveView(viewAllItemsViewModel.getViewName());
+                    viewManagerModel.firePropertyChanged();
+                }
+            }
+        });
+
+        nameInputField.addKeyListener(new KeyListener() {
+            @Override
+            public void keyTyped(KeyEvent e) {
+                GetClothingItemState currentState = getClothingItemViewModel.getState();
+                currentState.getClothingItem().setName(nameInputField.getText() + e.getKeyChar());
+                getClothingItemViewModel.setState(currentState);
+            }
+
+            @Override
+            public void keyPressed(KeyEvent e) {
+            }
+
+            @Override
+            public void keyReleased(KeyEvent e) {
+            }
+        });
+
+        minTempInputField.addKeyListener(new KeyListener() {
+            @Override
+            public void keyTyped(KeyEvent e) {
+                GetClothingItemState currentState = getClothingItemViewModel.getState();
+                currentState.getClothingItem().setMinimumAppropriateTemperature(Integer.parseInt(minTempInputField.getText() + e.getKeyChar()));
```

Comment:
Here Integer.parseInt() method is going to throw an exception if the user inputs a minus sign.